docs/usecases/actor_remoting.md

```diff
@@ -0,0 +1,204 @@
+---
+id: usecases_actor_remoting
+title:  "Actor Remoting"
```

Comment:
This is amazing effort, but as said, it was caused by misunderstanding. I propose that apply the idea [here](https://github.com/zio/zio-actors/issues/19), and to provide some simpler examples here.

Reply:
I've misunderstood it but I was also thinking about writing such module for `zio-actors`. I will take this piece to present this idea in issue or gitter.

But still I will provide some other examples there.

Reply:
It would be very useful, and there's already an issue, i've referenced it in my comment, so take a look!